react/Viewer/PdfMobileViewer.spec.jsx

```diff
@@ -52,54 +51,34 @@ describe('PdfMobileViewer', () => {
     expect(getByRole('progressbar'))
   })
 
-  describe('error when downloading file', () => {
-    it('should show network error message on native app and browser', async () => {
-      ImageLoader.prototype.checkImageSource = jest.fn().mockRejectedValue() // fail to load image
-
-      const { root } = setup({ file })
-      const { queryByRole, getByText } = root
-
-      const isMobileAppValues = [true, false]
-
-      for (const isMobileAppValue of isMobileAppValues) {
-        isMobileApp.mockReturnValue(isMobileAppValue)
-
-        await wait()
-
-        expect(queryByRole('progressbar')).toBeFalsy()
-        expect(
-          getByText(
-            'This file could not be loaded. Do you have a working internet connection right now?'
-          )
-        )
-      }
-    })
-  })
-
-  describe('error if file as no preview', () => {
+  describe('errors if file as no preview or failed to download', () => {
```

Comment:
Now, the test doesn't test the error case, right?

Reply:
C'est mon deuxième point dans la description de la PR

Reply:
We can still test if the noViewer is displayed, no? 

OK understood, https://github.com/cozy/cozy-ui/blob/fcd1b75a7fde1f6adc11d70f2de8e5125a1d30d6/react/Viewer/PdfMobileViewer.spec.jsx#L61 and https://github.com/cozy/cozy-ui/blob/fcd1b75a7fde1f6adc11d70f2de8e5125a1d30d6/react/Viewer/PdfMobileViewer.spec.jsx#L74 are testing that the NoViewer is displayed, right? Not intuitive at first

Reply:
Oui c'est les cas d'erreur ça effectivement